core/Symbols.cc

```diff
@@ -273,6 +274,18 @@ TypePtr ArgInfo::argumentTypeAsSeenByImplementation(Context ctx, core::TypeConst
 
 void Symbol::addMixin(const GlobalState &gs, SymbolRef sym) {
```

Comment:
Can you make this return either nothing or some error marked nodiscard, and handle reporting an error in the two places where `addMixin` is called? You will have a lot more information in scope to give a much better error message.

Reply:
What's `nodiscard`?

Reply:
Is there an existing place where we do this? `dispatchCall` defers reporting errors, but they aren't mutated w/ extra info once completed.

Reply:
We discussed online; `[[nodescript]]` is a function attribute in C++. I've made the requested change.